cli_tools_tests/e2e/variables.go

```diff
@@ -0,0 +1,53 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package e2e
+
+const (
+	projectIDWithoutDefaultServiceAccountFlag                      = "project_id_without_default_service_account"
+	projectIDWithoutDefaultServiceAccountPermissionFlag            = "project_id_without_default_service_account_permission"
+	computeServiceAccountWithoutDefaultServiceAccountFlag          = "compute_service_account_without_default_service_account"
+	computeServiceAccountWithoutDefaultServiceAccountPermissionFlag = "compute_service_account_without_default_service_account_permission"
```

Comment:
I'm finding these phrases bit puzzling. Perhaps to clarify what's contained in these fields, you could define a struct, and then instantiate it for each of your test projects?

Quick example:

```
type project struct {
  description string
  projectID string
  accountForComputeService string
  isComputeServiceDefaultAccount bool
}
```

Reply:
created a new struct ServiceAccountTestVariables